eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -354,8 +329,202 @@ private static void debug(final String msg) {
     /**
      * Padding according to current {@link #NESTING} level.
      * @return Padding string.
+     * @since 0.1
      */
     private static String padding() {
-        return String.join("", Collections.nCopies(NESTING.get(), "·"));
+        return String.join("", Collections.nCopies(PhDefault.NESTING.get(), "·"));
+    }
+
+    /**
+     * Make named attribute.
+     *
+     * @since 0.1
+     */
+    private static class NamedAttr {
```

Comment:
@l3r8yJ Can be just `Named`

Reply:
@l3r8yJ Also what is the point of replacing private function with new class? The object seems not to be part of any hierarchy and just replaces single function.

Reply:
@mximp I replace because [each private static method is a candidate for a new class](https://www.yegor256.com/2017/02/07/private-method-is-new-class.html).
And instead of calling it, we create a `new` instance of object.
what do you think?